panda/src/nativenet/socket_address.cxx

```diff
@@ -28,6 +28,13 @@ set_host(const std::string &hostname, unsigned short port) {
 
   struct addrinfo hints, *res = nullptr;
   memset(&hints, 0, sizeof(hints));
+
+// For some reason in msys this macro are not defined correctly in the
+// header file ws2tcpip.h
+#ifndef AI_ADDRCONFIG
+#define AI_ADDRCONFIG 0x00000400
+#endif
```

Comment:
Please move defines to the top of the file (under the includes), and not mid-code.  Same with the other block.